tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -3,18 +3,33 @@
 
 use Elementor\Core\Experiments\Manager as Experiments_Manager;
 use Elementor\Core\Upgrade\Manager;
+use Elementor\Tests\Phpunit\Elementor\Core\Experiments\Mock\Modules\Module_A;
+use Elementor\Tests\Phpunit\Elementor\Core\Experiments\Mock\Modules\Module_B;
 use ElementorEditorTesting\Elementor_Test_Base;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
 
 class Test_Manager extends Elementor_Test_Base {
+	/**
+	 * @var \Elementor\Core\Experiments\Manager
+	 */
+	private $experiments;
+
+	private $originalFeatures = [];
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->experiments = $this->elementor()->experiments;
+		$this->originalFeatures = $this->experiments->features;
+	}
 
 	public function tearDown() {
 		parent::tearDown();
 
-		$this->elementor()->experiments->remove_feature( 'test_feature' );
+		$this->experiments->features = $this->originalFeatures;
```

Comment:
Just re-initialize it, something like `$this->elementor()->experiments = new Experiments()`

Reply:
Handled.